modules/improvedigitalBidAdapter.js

```diff
@@ -0,0 +1,408 @@
+var LIB_VERSION_GLOBAL = '3.0.2';
+
+var CONSTANTS = require('src/constants');
+var utils = require('src/utils');
+var bidfactory = require('src/bidfactory');
+var bidmanager = require('src/bidmanager');
+var adloader = require('src/adloader');
+var Adapter = require('src/adapter').default;
+var adaptermanager = require('src/adaptermanager');
+
+const ImproveDigitalAdapter = function () {
+  var IMPROVE_DIGITAL_BIDDER_CODE = 'improvedigital';
```

Comment:
The bidder code is also used in the `createNew` function. Maybe extracting this into a `const` would be a good idea.